src/components/TradingviewChart/index.js

```diff
@@ -166,14 +177,19 @@ const TradingViewChart = ({
 
       // get the title of the chart
       function setLastBarText() {
-        toolTip.innerHTML =
-          `<div style="font-size: 16px; margin: 4px 0px; color: ${textColor};">${title} ${
-            type === CHART_TYPES.BAR && !useWeekly ? '(24hr)' : ''
-          }</div>` +
-          `<div style="font-size: 22px; margin: 4px 0px; color:${textColor}" >` +
-          formattedNum(base ?? 0, true) +
-          `<span style="margin-left: 10px; font-size: 16px; color: ${color};">${formattedPercentChange}</span>` +
-          '</div>';
+        toolTip.innerHTML = `
+        <div style="font-size: 16px; margin: 4px 0px; color: ${textColor};">
+        ${title} 
+        ${type === CHART_TYPES.BAR && !useWeekly ? '(24hr)' : ''}
+        </div>
+          <div style="font-size: 22px; margin: 4px 0px; color:${textColor}; display: flex; align-items: center; align-content: center;" >
+          ${
+            base
+              ? formattedNum(base ?? 0, true)
+              : `<div style="width: 35px; margin-left: 16px"><div class="dot-flashing"></div></div>`
+          }
+          <span style="margin-left: 10px; font-size: 16px; color: ${color};">${formattedPercentChange}</span>
```

Comment:
I guess here we can also hide the percentage while the data are still loading.
![image](https://user-images.githubusercontent.com/9011637/163793903-0c90873f-3163-49ed-a177-3de5bf147b14.png)